server/models/articleVotes.js

```diff
@@ -0,0 +1,43 @@
+export default (sequelize, DataTypes) => {
+  const ArticleVote = sequelize.define(
+    'ArticleVote',
+    {
+      id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        allowNull: false,
+        defaultValue: DataTypes.UUIDV4
+      },
+      vote: {
+        type: DataTypes.ENUM(1, -1, 0),
+        allowNull: false
+      },
+      createdAt: {
+        type: DataTypes.DATE
+      },
+      updatedAt: {
+        type: DataTypes.DATE
+      }
+    }
+  );
+
+  ArticleVote.associate = (models) => {
+    ArticleVote.belongsTo(
+      models.Article,
+      {
+        foriegnKey: 'articleId',
```

Comment:
it should be "foreignKey" not "foriegnKey"

Reply:
Okay. Will push a fix ASAP.